src/AddToDoForm.js

```diff
@@ -1,10 +1,17 @@
 import React from "react";
 
-function AddToDoForm() {
+function AddToDoForm(props) {
+  function handleAddTo(event) {
+    event.preventDefault();
+    let todoTitle = event.target.title.value;
```

Comment:
When are setting a value that does not change, `const` is preferred over `let`. (remember that event is an object so the values inside of it are still mutable even when stored in `const`)